app/e2e/tests/landing/landing-carousel.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('carousel buttons', () => {
+    test.use({ storageState: undefined });
```

Comment:
I think this can be removed now that we don't have a single storage state set in the playwright config. This was only necessary before when a login state was re-used across all tests, but now they are only used depending on the page class.